terraform/telegraf/lb.tf

```diff
@@ -72,6 +87,19 @@ resource "aws_lb_listener" "front_end" {
   }
 }
 
+resource "aws_lb_listener" "front_end2" {
+  load_balancer_arn = "${aws_lb.lb.arn}"
+  port              = "80"
```

Comment:
I assume you are using port 80 because port 443 is already in use but I think it would be more appropriate just use port 1619 to match what it actually is.  AJ can make an exception for that port on the sec tests.

Reply:
I had changed this because I don't like having something external (security exception) that needs configured, but it does feel wrong to use port 80 with ssl.
I'll switch it and ask AJ to add the exception.